backend/src/types/DataTypes.d.ts

```diff
@@ -114,3 +114,12 @@ export type DBDevPortfolioSubmission = {
   openedPRs: PullRequestSubmission[];
   reviewedPRs: PullRequestSubmission[];
 };
+
+export type DevPortfolioSubmissionRequestLog = {
+  timestamp: string;
+  email: string;
+  body: {
+    submission: DevPortfolioSubmission;
```

Comment:
Should this be `DBDevPortfolioSubmission`? Right now the entire member object is being sent to the DB instead of a doc reference. We could technically also remove the `member` field of a submission, since we've already associated the submission with an `email`.

Reply:
Good point, I'll change the type.